pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[tool.poetry]
+name = "smile-id-core"
+version = "1.0.8"
```

Comment:
you should grab this from __init__.py. This is a good thread https://github.com/python-poetry/poetry/issues/144